app/views/shop/_messages.html.haml

```diff
@@ -10,7 +10,7 @@
              register: ('<a auth="signup">' + t('.register') + '</a>').html_safe}
         - else
           = t '.require_customer_html',
-            {contact: ('<a href="##contact">' + t('.contact') + '</a>').html_safe,
+            {contact: (link_to t('.contact'), "#/contact", method: :get),
```

Comment:
Thanks @levent. The method: :get was added as a hack to force a page refresh so that the Contact details display. Otherwise the link does nothing, creating big problems from enterprise users.
Would love a better suggestion, less hacky, if you have one?